### PR TITLE
import-azure-core

### PR DESCRIPTION
**Pull request by changlong-liu**
For issue https://github.com/Azure/autorest.python/issues/996

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +20,14 @@ To install:
 Release History
 ---------------
 
+2022-05-05 Version 0.7.0
```
Comment (iscai-msft) [inline README.rst:23]: can you add a test for the serialization and deserialization errors to make sure we're throwing azure.core.exceptions.SerializationError and azure.core.exceptions.DeserializationError?

Comment (changlong-liu) [reply README.rst:23]: yes, I see there are many existing testcase around SerializationError/DeserializationError in test_serialization.py.  So I added a test to make sure msrest.exceptions.SerializationError is the same one in azure.core.exceptions.
https://github.com/Azure/msrest-for-python/pull/247/files#diff-7f5a7271c4eef44b2f5fda6334ecae0952550de169520dde4a919d86949361c2R2585

File: tests/test_serialization.py [hunk 2]
```diff
@@ -42,6 +42,7 @@
 from msrest.serialization import Model, last_restapi_key_transformer, full_restapi_key_transformer, rest_key_extractor
 from msrest import Serializer, Deserializer
 from msrest.exceptions import SerializationError, DeserializationError, ValidationError
```
Comment (msyyc) [inline tests/test_serialization.py:44]: `SerializationError` and `DeserializationError` definition is removed by the PR so we should remove it?

**Review by msyyc — APPROVED** [review]

**Review by iscai-msft — APPROVED** [review]

**Review by lmazuel — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (gdsuser1992) [reply README.rst:23]: We are creating a flask app. It is working fine in local environment (no version conflict issues).  Recently, when we tried pushing the code to azure app service, after successful build, when we access the webpage, its giving a 502 error. 

We tried to identify the issue and when we checked the logstream, the following error was found :
```

2022-06-16T12:28:03.580219773Z     from .universal_http.requests import (
2022-06-16T12:28:03.580223173Z   File "/tmp/8da4ea7dc4b7018/antenv/lib/python3.7/site-packages/msrest/universal_http/__init__.py", line 53, in <module>
2022-06-16T12:28:03.580226973Z     from ..exceptions import ClientRequestError, raise_with_traceback
2022-06-16T12:28:03.580230373Z   File "/tmp/8da4ea7dc4b7018/antenv/lib/python3.7/site-packages/msrest/exceptions.py", line 31, in <module>
2022-06-16T12:28:03.580234173Z     from azure.core.exceptions import SerializationError, DeserializationError
2022-06-16T12:28:03.580237773Z ImportError: cannot import name 'SerializationError' from 'azure.core.exceptions' (/tmp/8da4ea7dc4b7018/antenv/lib/python3.7/site-packages/azure/core/exceptions.py)
```

Are we facing any version conflict issues with azure-core and the msrest ? 

We are not adding the msrest package in requirements.txt and its loaded by default when we publish it to the app service. 

We tried to change the version of azure-core to 1) the latest version 2) 1.9.0  3) 1.11.0 - All these still end up giving the same error. 

Is there any work around for this issue ?? Please help.

Comment (gdsuser1992) [reply README.rst:23]: the azure-core version that we are using at present is 1.24.0

Comment (changlong-liu) [reply README.rst:23]: Hi @gdsuser1992 , latest msrest get worked with azure-core >=1.24.0.
Would you please make sure the azure-core at `/tmp/8da4ea7dc4b7018/antenv/lib/python3.7/site-packages/azure/core/` does be 1.24.0 or 1.24.1?

**Pull request merged**